gno.land/pkg/integration/doc.go

```diff
@@ -17,6 +17,10 @@
 //   - `--remote`, `--insecure-password-stdin`, and `--home` flags are set automatically to
 //     communicate with the gnoland node.
 //
+// 3. `adduser`:
+//   - Creates a new user in the default keybase directory
```

Comment:
Period (Just to match with existing style)
```suggestion
//   - Creates a new user in the default keybase directory.
```